examples/sdk-examples/src/examples/dateFilter/DateFilterComponentExample.tsx

```diff
@@ -196,6 +196,7 @@ export const DateFilterComponentExample: React.FC = () => {
                 availableGranularities={availableGranularities}
                 customFilterName="Selected date"
                 dateFilterMode="active"
+                dateFormat="MM/dd/yyyy"
```

Comment:
should we use the constant DEFAULT_DATE_FORMAT declared in libs/sdk-ui-filters/src/DateFilter/DateFilter.tsx here

Reply:
I don't think so, examples are used for learning the usage of our library (by customer or QA?).
We should use format directly

Reply:
Agreed